app/presenters/two_factor_authentication/webauthn_platform_selection_presenter.rb

```diff
@@ -5,7 +5,10 @@ def method
     end
 
     def render_in(view_context, &block)
-      view_context.render(WebauthnInputComponent.new(platform: true), &block)
+      view_context.render(
+        WebauthnInputComponent.new(platform: true, passkey_supported_only: configuration.blank?),
+        &block
```

Comment:
would it be worth adding any spec coverage for this presenter's `configuration.blank?` logic?